code/HUXt.py

```diff
@@ -1403,9 +1540,26 @@ def solve_upwind(model, v_input):
     :param v_input: Time series of inner boundary solar wind speeds, in km/s.
     :return:
     """
+    
+    #check the CFL condition is met
+    constants= huxt_constants()
+    alpha = constants['alpha']  # Scale parameter for residual SW acceleration
+    r_accel = constants['r_accel']  # Spatial scale parameter for residual SW acceleration
+    v_max = constants['v_max']
```

Comment:
These are already available as attributes of model, so it's probably easier just to refer to them there?